it/helpers/TestITData.scala

```diff
@@ -29,6 +29,7 @@ import uk.gov.hmrc.taxenrolmentassignmentfrontend.models._
 object TestITData {
 
   val NINO: String = "JT872173A"
+  val mismatchNino: String = "testFakeNino" // TODO - Add NINO generator
```

Comment:
comment?

Reply:
No need for a generator but a random nino

Reply:
Done